react_project/src/page/Market.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react'
+import BestProduct from '../components/BestProduct'
+import AllProducts from '../components/AllProducts'
+import { getProducts } from '../data/api';
+
+
+export default function Market() {
+  const [order, setOrder] = useState('createdAt');
+  const [products, setProducts] = useState([]);
+  const sortedProducts = products.sort((a, b) => b[order] - a[order]);
+
+  const handleNewstClick = () => setOrder('createdAt');
+  const handleFavoriteClick = () => setOrder('favorite');
+  const handleLoad = async () => {
+  const { list } = await getProducts();
+    setProducts(list);
+  }
+
+  useEffect(() => {
+    handleLoad();
+  }, []);
+
+  return (
+    <>
+      <BestProduct products={sortedProducts} order='favorite' />
```

Comment:
```js
export default function BestProduct({ products }) {
```
마찬가지로 BestProduct 또한 order가 없기 때문에 아래처럼 할 수 있을듯합니다~
```suggestion
      <BestProduct products={sortedProducts} />
```